### DISCUSSION TITLE
Support legacy RegionOfflineManager

### DISCUSSION THREAD

**Discussion by kristfal** [rnmapbox/maps#3028] [Ideas]
Hey,

we are about to upgrade to V10, but need [support for RegionOfflineManager](https://github.com/mapbox/mapbox-maps-ios/issues/546) as we run on custom tiles. 

Plan to contribute to this ourselves. @mfazekas you good with an approach of building a 1:1 mirror of the OfflineManager API named RegionOfflineManager which uses the legacy APIs internally?

**Comment by mfazekas** [comment]
It’s sounds like a plan. We might think a bit about naming - like LegacyOfflineManager or ClassicOfflineManager or SelfHostedOfflineManager but not sure
